bluemira/utilities/tools.py

```diff
@@ -299,6 +300,15 @@ def cross(
 # =====================================================
 
 
+def floatify(x: npt.ArrayLike):
+    """
+    Avoid numpy warnings for float(x) for >0 rank scalars
+    """
+    if x is None:
+        raise TypeError("argument must be a string or a real number, not 'NoneType'")
```

Comment:
The argument could be an array too no?

Maybe just "The argument cannot be None" or something